### ISSUE TITLE
test-benchmark

### ISSUE THREAD

**Issue by freinkxhihani-a11y** [bazelbuild/bazel#30056]

**Comment by bharadwaj08-one** [comment]
@freinkxhihani-a11y  We are closing this issue since it doesn't have valid details. Please tag `@bazelbuild/triage` if you have any questions or want this reopened.
